.github/workflows/npmpublish.yml

```diff
@@ -38,10 +50,10 @@ jobs:
       - name: Install
         run: npm ci
 
-      - name: Build
-        run: npm run build:prod
+      - name: Production Build
+        run: npm run build:prod --if-present
```

Comment:
We know it is present, is not better to fail if is not ?

Reply:
It's better to fail, I will remove the CLI options.